src/views/FlowNotify.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+	<div>
+		<input type="text"
+			max-length="80"
+			:placeholder=placeholder
+			:value=currentInscription
+			@input="(newValue) => newValue !== null && $emit('input', JSON.stringify({ inscription: newValue.target.value } ))" />
```

Comment:
Would be good to move that to a dedicated method.

Reply:
addressed